Namsonx/task/stabi branch

namsonx opened the pull request.

HolQue committed:
Merge branch 'develop' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
Merge branch 'develop' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
Merge branch 'develop' into HolQue/task/selftest_and_documentation_extensions

namsonx committed:
Updated error message based on ticket 117

namsonx committed:
Merge branch 'develop' into namsonx/task/stabi_branch

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

test-fullautomation reviewed (changes requested):
Hi Son,
thank you for your pull-request.
I merged https://github.com/test-fullautomation/robotframework-testsuitesmanagement/pull/222/files into you branch.
Please check and create a pull-request update when all runs properly.
Thank you,
 Thomas

namsonx committed:
Updated pattern when handling a key name during update nested parameter.

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

HolQue reviewed (changes requested):
Error message still not OK.

https://github.com/test-fullautomation/python-jsonpreprocessor/issues/111

namsonx committed:
Tiny change regarding JSON file path computation based on ticket 152

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
Added GitHub link to JSON file in HTML test case list

namsonx committed:
Updated history for version 0.3.0

namsonx committed:
Fixed parameter substitution issue when colon inside string ticket_92

HolQue committed:
JPP_1000 extended (implicit creation combinations)

Because of
https://github.com/test-fullautomation/python-jsonpreprocessor/issues/153
extensions partially commented out

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
JPP_0505 activated because of solved issue

HolQue committed:
JPP_1150 added (cyclic imports)

namsonx committed:
Tiny change to update error message log, ticket 78

HolQue committed:
Merge remote-tracking branch 'origin/namsonx/task/stabi_branch' into HolQue/task/selftest_and_documentation_extensions

HolQue committed:
JPP_1151 added (cyclic imports)

HolQue committed:
JPP_1200 added (relative path to JSON file)

test-fullautomation committed:
Merge pull request #154 from test-fullautomation/HolQue/task/selftest_and_documentation_extensions

Self test extensions

test-fullautomation reviewed (approved):
Hi Son,
looks good to me!
Thank you,
 Thomas
